umbral/pre.py

```diff
@@ -85,11 +85,14 @@ def from_bytes(cls, capsule_bytes: bytes, curve: ec.EllipticCurve = None):
         return cls(point_eph_e=eph_e, point_eph_v=eph_v, bn_sig=sig,
                    e_prime=e_prime, v_prime=v_prime, noninteractive_point=eph_ni)
 
+    def _original_to_bytes(self) -> bytes:
```

Comment:
Look at the private methods on @cygnusv!  That's right, the `bytes().join()`!

Reply:
I don't get your comment ^_^U 
Is that a private joke?

Reply:
It's an allusion to this famous scene from Pulp Fiction.

https://youtu.be/Hrm-rPSCIBw

Reply:
Oh right...but poor Brad ended dead :(

Reply:
They also shot Marvin in the face.. :'(

RIP Marvin